Policies/Kubernetes/allowed-host-paths/rego/src_test.rego

```diff
@@ -0,0 +1,243 @@
+package k8sazurehostfilesystem
+
+test_input_hostpath_allowed_all {
+    input := { "review": input_review, "parameters": input_parameters_empty}
+    results := violation with input as input
+    count(results) == 0
+}
+test_input_hostpath_allowed_readonly {
+    input := { "review": input_review_many, "parameters": input_parameters_in_list}
+    results := violation with input as input
+    count(results) == 0
+}
+test_input_hostpath_not_allowed_readonly {
+    input := { "review": input_review, "parameters": input_parameters_not_in_list}
+    results := violation with input as input
+    count(results) == 1
+}
+test_input_hostpath_many_not_allowed_readonly {
+    input := { "review": input_review_many, "parameters": input_parameters_not_in_list}
+    results := violation with input as input
+    count(results) == 2
+}
+test_input_hostpath_allowed_writable_not_allowed {
+    input := { "review": input_review_writable, "parameters": input_parameters_in_list}
+    results := violation with input as input
+    count(results) == 1
+}
+test_input_hostpath_not_allowed_writable {
+    input := { "review": input_review_writable, "parameters": input_parameters_not_in_list}
+    results := violation with input as input
+    count(results) == 1
+}
```

Comment:
Can you add tests with input_review_writable and input_parameters_in_list_writable?